contracts/attrs/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "attrs"
-version = "1.1.2"
+version = "1.2.0"
 authors = ["David Pederson <dpederson@figure.com>", "Ken Talley <ktalley@figure.com>"]
```

Comment:
You could fix this...

Reply:
I'd like to have the versions inherit from the root project, but for now I've just manually set all subproject's version to the same as root. Or was it something else?

Reply:
My comment is on the next line (`authors`)